frontend/shared/src/components/hocs/withAuthRedirect.tsx

```diff
@@ -1,10 +1,11 @@
 import { useRouter } from 'next/router';
 import React from 'react';
+import PageLoadingSpinner from 'shared/components/pages/PageLoadingSpinner';
 import useAuth from 'shared/hooks/useAuth';
 import isServerSide from 'shared/server/is-server-side';
 
 const DefaultLoadingFallback = (): React.ReactElement<unknown> => (
-  <p>Uudelleenohjataan...</p>
+  <PageLoadingSpinner />
 );
```

Comment:
What's the purpose of this component? 
It looks like `DefaultLoadingFallback` is assigned to `LoadingComponent = DefaultLoadingFallback`; why don't you assign `LoadingComponent = PageLoadingSpinner` instead?

Reply:
Definitely, coding too fast without thinking :)

Reply:
fixed